adapters/taglib/taglib.go

```diff
@@ -122,7 +122,7 @@ func parseTIPL(tags map[string][]string) {
 	addRole := func(currentRole string, currentValue []string) {
 		if currentRole != "" && len(currentValue) > 0 {
 			role := tiplMapping[currentRole]
-			tags[role] = append(tags[currentRole], strings.Join(currentValue, " "))
+			tags[role] = append(tags[role], strings.Join(currentValue, " "))
```

Comment:
Nice catch! To cover this situation, please add another DJ-mix to the test so we are sure it is appending to the right slice

Reply:
So, 
```go
tags["tipl"] = []string{"arranger Andrew Powell DJ-mix François Kevorkian DJ-mix Jane Doe engineer Chris Blair"}
```
?